libs/klv/src/main/java/org/codice/alliance/libs/klv/Utilities.java

```diff
@@ -25,4 +33,36 @@ public static boolean isEmptyString(Serializable serializable) {
     public static boolean isNotEmptyString(Serializable serializable) {
         return !isEmptyString(serializable);
     }
+
+    static void safelySetAttribute(Metacard metacard, Attribute attribute) {
+        safelySetAttribute(metacard, attribute.getName(), attribute.getValues());
+    }
+
+    static void safelySetAttribute(Metacard metacard, String attributeName, Serializable value) {
+        notNull(metacard, "");
```

Comment:
For these `notNull` statements, it would be beneficial to use add the message describing what parameter was null.  i.e. `notNull(metacard, "Metacard cannot be null.`);